dialogue-core/src/main/java/com/palantir/dialogue/core/DialogueTracing.java

```diff
@@ -57,7 +59,7 @@
                 sink.tag(target, tags);
                 int status = response.code();
                 sink.tag(target, "outcome", status / 100 == 2 ? "success" : "failure");
-                sink.tag(target, "status", Integer.toString(status));
+                sink.tag(target, TraceTags.HTTP_STATUS_CODE, Integer.toString(status));
```

Comment:
You could probably also centralize the int -> String for status codes? I think you have that multi-if-statement in abunch of places.